### PR TITLE
Add --force-expire option

### LINKED ISSUE #10: Bypass GNOME's "no expiration" design decision
GNOME made the (pretty weird) "design decision" to always ignore the expire time of notifications and even making all notifications persistent (the latter since the introduction of GNOME's notification center). Most applications bypass this behaviour by manually closing notifications after some individual timeouts (yes, instead of unifying expiration times, GNOME made it worse).

It would be great if `notify-send.sh` could easily bypass this behaviour, too. I wrote a simple wrapper script, even though implementing this into `notify-send.sh`'s base script would obviously be simpler and a better approach:

```bash
#!/bin/bash
EXPIRE_TIME=-1
PRINT_ID=

NOTIFY_ARGS=()
while (( $# > 0 )); do
    case "$1" in
        -t|--expire-time|--expire-time=*)
            [[ "$1" = --expire-time=* ]] \
                && EXPIRE_TIME="${1#*=}" \
                || { shift; EXPIRE_TIME="$1"; }
            NOTIFY_ARGS+=( --expire-time "$EXPIRE_TIME" )
            ;;

        -p|--print-id)
            PRINT_ID=yes
            NOTIFY_ARGS+=( --print-id )
            ;;

        *)
            NOTIFY_ARGS+=( "$1" )
            ;;
    esac
    shift
done

NOTIFY_SEND="$(dirname "${BASH_SOURCE[0]}")/notify-send.sh"
if (( "$EXPIRE_TIME" != -1 )); then
    NOTIFICATION_ID="$("$NOTIFY_SEND" --print-id "${NOTIFY_ARGS[@]}")"
    NOTIFY_RC=$?

    if [[ -n "$PRINT_ID" ]]; then
        echo "$NOTIFICATION_ID"
    fi

    if (( "$EXPIRE_TIME" > 0 )); then
        SLEEP_TIME="$(bc <<< "scale=3; $EXPIRE_TIME / 1000")"
        ( sleep "$SLEEP_TIME" ; "$NOTIFY_SEND" --close "$NOTIFICATION_ID" ) &
    else
        "$NOTIFY_SEND" --close "$NOTIFICATION_ID"
    fi

    exit $NOTIFY_RC
fi

"$NOTIFY_SEND" "${NOTIFY_ARGS[@]}"
exit $?
```

### PR DESCRIPTION
**Pull request by PhrozenByte**
Resolves #10

### CODE REVIEW

File: notify-send.sh [hunk 1]
```diff
@@ -122,6 +123,11 @@ notify() {
         echo "$NOTIFICATION_ID"
     fi
 
+    if [[ -n "$FORCE_EXPIRE" ]] ; then
+        SLEEP_TIME="$(bc <<< "scale=3; $EXPIRE_TIME / 1000")"
```
Comment (vlevit) [inline notify-send.sh]: I'm not fully sure  bc is installed on all systems where notify-send.sh is used and I prefer avoiding unnecessary hard dependency, soft dependcy is ok to me though. So I just would add a run-time check and print a friendly error message about bc not found, something like this:

```type bc &> /dev/null || { echo "bc command not found. Please installed bc package."; exit 1;  }```

**Comment by vlevit** [comment]
Thank you @PhrozenByte. Let's add a check for bc command availability and I will merge it.

**Comment by PhrozenByte** [comment]
:+1:

**Comment by vlevit** [comment]
@PhrozenByte Thanks!

**Pull request merged**